fontforge/splineutil.c

```diff
@@ -7953,3 +7953,19 @@ bigreal DistanceBetweenPoints( BasePoint *p1, BasePoint *p2 )
     t = sqrt( t );
     return t;
 }
+
+int SplineSetNumberPoints(SplineSet *ss, int order2) {
+    int pnum=0;
+    if ( order2 ) {
+        return SSTtfNumberPoints(ss);
```

Comment:
Seems like this clause is a better place for `SplinePointListCopy1()` given that we don't need it for the cubic case. Might even be better to make a `SSTtfNumberPoints1` with its "innards" and that takes a boolean to control whether it modifies its argument. Then `SSTtfNumberPoints()` can just call that for each SS with `false` and you can call it here with `true` (or the other way around -- I don't have any intuition for the boolean polarity).